src/widget/wtracktableview.cpp

```diff
@@ -1074,6 +1087,49 @@ void WTrackTableView::contextMenuEvent(QContextMenuEvent* event) {
             }
         }
         m_pMenu->addMenu(m_pBPMMenu);
+
+        // Track color menu only appears if at least one track is selected
+        if (indices.size()) {
+            // Get color of first selected track
+            int column = trackModel->fieldIndex(LIBRARYTABLE_COLOR);
+            QModelIndex index = indices.at(0).sibling(indices.at(0).row(), column);
+            mixxx::RgbColor::optional_t trackColor = std::nullopt;
+            if (!index.data().isNull()) {
+                trackColor = mixxx::RgbColor::optional(index.data().toUInt());
+            }
+
+            // Check if all other selected tracks have the same color
+            for (int i = 1; i < indices.size(); ++i) {
+                int row = indices.at(i).row();
+                QModelIndex index = indices.at(i).sibling(row, column);
+
+                mixxx::RgbColor::optional_t otherTrackColor = std::nullopt;
+                if (!index.data().isNull()) {
+                    otherTrackColor = mixxx::RgbColor::optional(index.data().toUInt());
+                }
+
+                if (trackColor != otherTrackColor) {
+                    trackColor = std::nullopt;
+                    break;
+                }
+            }
+
+            PredefinedColorPointer predefinedTrackColor = nullptr;
+            if (trackColor) {
+                for (PredefinedColorPointer color : Color::kPredefinedColorsSet.allColors) {
+                    if (mixxx::RgbColor(color->m_defaultRgba.rgb()) == trackColor.value()) {
```

Comment:
```
src/widget/wtracktableview.cpp:1120:83: error: 'value' is unavailable: introduced in macOS 10.14
                    if (mixxx::RgbColor(color->m_defaultRgba.rgb()) == trackColor.value()) {
```

Use `*trackColor` instead of `trackColor.value()`